src/main/scala/mesosphere/marathon/api/v2/InfoEmbedResolver.scala

```diff
@@ -46,13 +46,30 @@ private[v2] object AppInfoEmbedResolver {
     }
 
     def separatePrefix(embedMe: String): (String, String) = {
-      val removablePrefix = EmbedAppsPrefixes.find(embedMe.startsWith(_)).getOrElse("")
+      val removablePrefix = EmbedAppsPrefixes.find(embedMe.startsWith).getOrElse("")
```

Comment:
do you really like that better? With Connor I agreed that we mostly like the `(_)` syntax better. That syntax has also the advantage of working with additional default arguments. (Which doesn't matter with "startsWith" admittedly)